<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>soldwindow</class>
 <widget class="QWidget" name="soldwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sold</string>
  </property>
  <widget class="QTableWidget" name="soldWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>751</width>
     <height>391</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>187</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>37</number>
   </attribute>
   <column>
    <property name="text">
     <string>商品</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>售价</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>库存</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>出售件数</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="sold_backbtn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sold_btn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认出售</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sold_backbtn</sender>
   <signal>clicked()</signal>
   <receiver>soldwindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>706</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
